src/settings/index.js

```diff
@@ -1,26 +1,32 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { FormattedMessage } from 'react-intl';
+
 import { Settings } from '@folio/stripes/smart-components';
-import GeneralSettings from './general-settings';
-import SomeFeatureSettings from './some-feature-settings';
 
-export default class OrdersSettings extends React.Component {
-  pages = [
-    {
-      route: 'general',
-      label: <FormattedMessage id="ui-orders.settings.general" />,
-      component: GeneralSettings,
-    },
-    {
-      route: 'somefeature',
-      label: <FormattedMessage id="ui-orders.settings.some-feature" />,
-      component: SomeFeatureSettings,
-    },
-  ];
+import POLinesLimit from './POLinesLimit';
+
+class Orders extends Component {
+  constructor(props){
+    super(props)
+
+    this.pages = [
+      {
+        component: POLinesLimit,
+        label: <FormattedMessage id="ui-orders.settings.polines-limit" />,
+        route: 'polines-limit',
+      },
+    ];
+  }
 
   render() {
     return (
-      <Settings {...this.props} pages={this.pages} paneTitle="ui-orders" />
+      <Settings
+        {...this.props}
+        pages={this.pages}
+        paneTitle={<FormattedMessage id="ui-orders.settings.index.paneTitle" />}
+      />
     );
   }
 }
+
+export default Orders;
```

Comment:
`Orders` doesn't reflect the meaning of this component. It doesn't display orders or something. Maybe there is a better name. Actually, what's wrong with previous `OrdersSettings`?